### PR TITLE
WFLY-18428: Add get started page

### PR DESCRIPTION
**Pull request by hpehl**
https://issues.redhat.com/browse/WFLY-18428

Preview: https://hpehl.github.io/wildfly.org/

### CODE REVIEW

**Comment by hpehl** [comment]
@bstansberry @jmesnil, could you please review and give feedback? 

I used the content from Jeff's fork, removed everything related to the guides, and polished some CSS styles. I verified the steps on the getting started page, but it would be great if you also confirm that it is working as expected. I hope I didn't forget anything. 

@bstansberry, Jeff mentioned that the link to the getting started page in the header should be more prominent. What do you have in mind?

**Review by jamezp — APPROVED** [review]

**Comment by hpehl** [comment]
@bstansberry, @jamezp, @jmesnil, please wait until you merge this PR. I'd like to record a short video about the new getting started experience and add a vlog entry to the news sections. I think it would make sense if both PRs would be merged simultaneously.

File: getstarted.adoc
Comment (fjuma) [inline getstarted.adoc:50]: Just minor, s/run/runs

File: _data/projectfooter.yml
Comment (fjuma) [inline _data/projectfooter.yml:11]: Just wondering if the URL should be /get-started instead of /getstarted. Took a quick look at quarkus.io and they are using /get-started.

File: getstarted.adoc
Comment (hpehl) [reply getstarted.adoc:50]: Well spotted! Thanks @fjuma

File: _data/projectfooter.yml
Comment (hpehl) [reply _data/projectfooter.yml:11]: I think `/get-started` is better. I'll fix that...

**Comment by fjuma** [comment]
> @bstansberry, Jeff mentioned that the link to the getting started page in the header should be more prominent. What do you have in mind?

Related to this, just wanted to add a comment here so we don't forget. When making the getting started page more prominent, it would also be good to figure out where to add the link to the "Getting Started on OpenShift" [guide](https://docs.wildfly.org/30/Getting_Started_on_OpenShift.html) so that can be easily reached from wildfly.org.

File: getstarted.adoc
Comment (bstansberry) [inline getstarted.adoc:12]: For now...

s/the latest LTS version/`17`/g

See https://www.wildfly.org/news/2023/10/18/WildFly30-Released/ and 'Recommended SE Version' for why.

In the future I want this to be driven by metadata in _data/releases.yml, but that's a separate task.

File: getstarted.adoc
Comment (bstansberry) [inline getstarted.adoc:37]: s/WildFly/the WildFly/g

File: getstarted.adoc
Comment (bstansberry) [inline getstarted.adoc:41]: s/command/commands

File: getstarted.adoc
Comment (bstansberry) [inline getstarted.adoc:49]: s/in WildFly/into WildFly/g

File: getstarted.adoc
Comment (bstansberry) [inline getstarted.adoc:64]: s/your/you

File: getstarted.adoc
Comment (bstansberry) [inline getstarted.adoc:68]: s/`wildfly:dev`/`wildfly:dev` goal/g

File: _layouts/index.html
Comment (bstansberry) [inline _layouts/index.html:15]: I think this should be first (to the left) with 'Get Started' in the middle and with color.  So 'Get Started' just replaces the old 'Download the Zip' in the central, highlighted spot. 'Download WildFly' is 2nd most prominent (on the left.)

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks @hpehl. Please ping me in zulip when you are ready.

**Comment by hpehl** [comment]
Thanks, @bstansberry, for the feedback and fixes. I've incorporated the changes and updated the preview at https://hpehl.github.io/wildfly.org/

File: _layouts/getstarted.html
Comment (bstansberry) [inline _layouts/getstarted.html:16]: Other pages don't have this and I don't think this one should. Just because you can go to this page from any other page, as it's is linked from the standard header/footer. So if you do that, this is not a 'Back' link, it's a 'Home' link.

Comment (hpehl) [reply _layouts/getstarted.html:16]: Yes, I agree. I'm going to change that to "Home"

**Review by bstansberry — APPROVED** [review]
Great! LGTM.

I've approved this but I'm putting the 'hold' label on this to give a you a chance to decide if you still want to [coordinate merging it with other publicity](https://github.com/wildfly/wildfly.org/pull/473#issuecomment-1793379175).

**Review by jmesnil — APPROVED** [review]
LGTM, thanks @hpehl

**Comment by hpehl** [comment]
@bstansberry, I don't know exactly when I'll finish the vLog. So IMO we should merge this PR now. Once I have finished the vLog, I'll create another PR for the blog post.

**Comment by bstansberry** [comment]
Awesome! Thanks @hpehl @jmesnil! This is a very big step!

**Pull request merged**
